portal-api/src/v1/gef/models/application.js

```diff
@@ -34,6 +34,7 @@ class Application {
       this.editedBy = editedBy;
       this.additionalRefName = req.additionalRefName ? String(req.additionalRefName) : null;
       this.facilitiesUpdated = null;
+      this.portalActivities = [];
```

Comment:
Do we have this covered in api test? i.e so when an application is created it has this empty array?